### PR TITLE
Fix example snippets Set.swift

### PR DESCRIPTION
**Pull request by valeriyvan**
Fix example snippets in Set.swift

### CODE REVIEW

File: stdlib/public/core/Set.swift [hunk 1]
```diff
@@ -116,14 +116,6 @@
 /// collection wrapper instead of a set. To restore efficient set operations,
 /// create a new set from the result.
 ///
-///     let morePrimes = primes.union([11, 13, 17, 19])
-///
-///     let laterPrimes = morePrimes.filter { $0 > 10 }
-///     // 'laterPrimes' is of type Array<Int>
-///
-///     let laterPrimesSet = Set(morePrimes.filter { $0 > 10 })
-///     // 'laterPrimesSet' is of type Set<Int>
```
Comment (xwu) [inline stdlib/public/core/Set.swift:125]: Why is this deleted?

Comment (valeriyvan) [reply stdlib/public/core/Set.swift:125]: Because it's not relevant anymore.
`Set` defines `filter` to return `Set`.
I fail to find in `Sequence` and `Collection` any good example to replace this snippet with.
There're thing like `prefix` which return `Array` but they all have little sense to apply to sets.

Comment (xwu) [reply stdlib/public/core/Set.swift:125]: I think `map` would be a good example. What do you think?

File: stdlib/public/core/Set.swift [hunk 2]
```diff
@@ -116,13 +116,12 @@
...
+///     let formatter = NumberFormatter()
+///     formatter.numberStyle = .spellOut
+///     let primesSpellOut = primes.map(NSNumber.init).compactMap(formatter.string)
+///     // 'primesSpellOut' is of type Array<String>
+///     let primesSpellOutSet = Set(primes.map(NSNumber.init).compactMap(formatter.string))
+///     // 'primesSpellOutSet' is of type Set<String>
```
Comment (xwu) [inline stdlib/public/core/Set.swift:124]: What do you think of sticking to simple standard library types? A user who would benefit from this example likely has not progressed to learning about NumberFormatter and NSNumber.

It does not have to a useful example; it merely has to demonstrate the return type of a function such as ‘map’ and how to get a Set back.

**Comment by valeriyvan** [comment]
done

**Comment by CodaFi** [comment]
@swift-ci smoke test and merge

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
⛵
